components/brave_new_tab_ui/components/default/bitcoinDotCom/index.tsx

```diff
@@ -18,6 +18,7 @@ interface State {
   currentAmount: string
   assetsShowing: boolean
   fiatCurrenciesShowing: boolean
+  amountError: boolean
```

Comment:
nit: if this were a string then you could support multiple types of error (`null` would mean no error), but I guess here there is only 1 invalid amount error case?

Reply:
Correct yeah, this is the only path to user error at the moment